vault/logical_system_test.go

```diff
@@ -4976,6 +5143,13 @@ func TestSystemBackend_Loggers(t *testing.T) {
 				t.Fatalf("unexpected error, err: %v, resp: %#v", err, resp)
 			}
 
+			schema.ValidateResponse(
+				t,
+				schema.FindResponseSchema(t, paths, 10, logical.ReadOperation),
```

Comment:
If it's not too much work, could you please switch these to `GetResponseSchema(t, ...b.Route(...))` to make them less brittle. See https://github.com/hashicorp/vault/pull/18865 for a better example.

Reply:
Ah I didn't know that was possible much much better and I updated everywhere.